sdk/storage/azure-storage-queue/samples/queue_samples_message.py

```diff
@@ -146,14 +146,45 @@ def send_and_receive_messages(self):
             # Delete the queue
             queue.delete_queue()
 
-    def delete_and_clear_messages(self):
+    def list_message_pages(self):
         # Instantiate a queue client
         from azure.storage.queue import QueueClient
         queue = QueueClient.from_connection_string(self.connection_string, "myqueue4")
 
         # Create the queue
         queue.create_queue()
 
+        try:
+            queue.send_message(u"message1")
+            queue.send_message(u"message2")
+            queue.send_message(u"message3")
+            queue.send_message(u"message4")
+            queue.send_message(u"message5")
+            queue.send_message(u"message6", visibility_timeout=30)
+
+            # [START receive_messages_listing]
+            # Store two messages in each page
+            messages = queue.receive_messages(messages_per_page=2).by_page()
```

Comment:
by_page() will give you pages of messages.

so we can make it more clear like 

```
pages_of_message = queue.receive_messages(messages_per_page=2).by_page()
for page_of_message in pages_of_message:
    for message in page_of_message:
        print(message)
```

Reply:
This example already exists in the sample `send_and_receive_messages` (above this new sample). I think the client wanted a sample that shows the ability to use `next()` to iterate through the messages that are in a page one by one. That is why I used `list` and `next` in this sample to show how we can iterate through individual messages in a given page.